src/main/java/com/internet/shop/dao/GenericDao.java

```diff
@@ -0,0 +1,16 @@
+package com.internet.shop.dao;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface GenericDao<T, V> {
+    T create(T element);
```

Comment:
```suggestion
    T create(T item);
```
keep your code consistent, if you use "item" in service, here should be same naming

Reply:
fixed